Makefile

```diff
@@ -37,6 +37,11 @@ csv:
 ddsidl:
 	${TOOLSDIR}/vspec2ddsidl.py -I ./spec ./spec/VehicleSignalSpecification.vspec vss_rel_$$(cat VERSION).idl
 
+# Verifies that supported overlay combinations are syntactically correct.
+overlays:
+	${TOOLSDIR}/vspec2csv.py -I ./spec -o overlays/profiles/motorbike.vspec ./spec/VehicleSignalSpecification.vspec vss_rel_$$(cat VERSION)_motorbike.idl
```

Comment:
Doesn't matter for the intended goal of checking "parsability" of the overlay, but test uses vspec2csv however  saves result  to .idl
ceterum censeo vspec2json ist the only tool that counts 😛